docs/book/src/user/quick-start.md

```diff
@@ -7,10 +7,51 @@ In this tutorial we'll cover the basics of how to use Cluster API to create one
 - Install and setup [kubectl] in your local environment.
 - A Kubernetes cluster to be used as [management cluster].
   For the purpose of this quick start we suggest you to use [kind] as a _non production-ready_ solution.
+
+{{#tabs name:"kind-cluster" tabs:"AWS,Docker,vSphere"}}
+{{#tab AWS}}
+
+  ```bash
+  kind create cluster --name=clusterapi
+  export KUBECONFIG="$(kind get kubeconfig-path --name="clusterapi")"
+  ```
```

Comment:
Not a blocker, but it would be nice if we could reference this block form other tabs rather than replicating in multiple places.

Reply:
I'd prefer this as well, maybe we can have a special KIND configuration section when you have to pick your infrastructure provider?